sepal_ui/sepalwidgets/app.py

```diff
@@ -457,16 +478,31 @@ def add_banner(self, msg, **kwargs):
         kwargs["type"] = kwargs.pop("type", "info")
         kwargs["border"] = kwargs.pop("border", "left")
         kwargs["class_"] = kwargs.pop("class_", "mt-5")
-        kwargs["transition"] = kwargs.pop("transition", "slide-x-transition")
+        kwargs["transition"] = kwargs.pop("transition", "scroll-x-transition")
         kwargs["prominent"] = kwargs.pop("prominent", True)
         kwargs["dismissible"] = kwargs.pop("dismissible", True)
         kwargs["children"] = [msg]  # cannot be overwritten
+        kwargs["_metadata"] = {"id_": id_}
+        kwargs["v_model"] = kwargs.pop("v_model", False)
+
+        # prepare a test to find an existing banner
+        def test(m):
+            m is not None and m.pop("id_", None) == id_
```

Comment:
You're missing a `return` here.
And if you click `theme`, then `locale`, and `theme` button again, new widgets will be displayed.

Reply:
that was unexpected but I found the feature nice in the end. Like as long as you click on the same btn we continue to refresh the alert, if you click elsewhere, we "forget" it and reset the alert again. that's my 2 cents, if I'm alone I agree to change it.

Reply:
Well... the nested if-solution was working pretty well tho, I'll let you chose the better one ;).

Reply:
too muche resonsability 😄

Reply:
I think I found it: 
- attributes is always a dictionary even when not set so I can directly use the `get` method without checking existence
- if `alert` is set then I remove it else I do nothing (the use of the `or` is more elegant and don't create indentation
- prevent the previous bug

Reply:
Perfect!! It worked pretty well, I won't bother you anymore with this!